<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>541</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MapDemo</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="decorLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="sceneLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionFloatSetting"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSavePlan">
   <property name="text">
    <string>保存方案</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionFloatSetting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>视图控制按钮</string>
   </property>
  </action>
  <action name="actionRadarDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>雷达部署</string>
   </property>
  </action>
  <action name="actionTargetDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>目标设置</string>
   </property>
  </action>
  <action name="actionSetCenter">
   <property name="text">
    <string>中心站设置</string>
   </property>
  </action>
  <action name="actionPlotDock">
   <property name="text">
    <string>juqing</string>
   </property>
  </action>
  <action name="actionClearAll">
   <property name="text">
    <string>清屏</string>
   </property>
  </action>
  <action name="actionAddRadar">
   <property name="text">
    <string>添加雷达</string>
   </property>
  </action>
  <action name="actionTargetArrange">
   <property name="text">
    <string>目标规划</string>
   </property>
  </action>
  <action name="actionId">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>批号</string>
   </property>
  </action>
  <action name="actionGeograph">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>地理坐标</string>
   </property>
  </action>
  <action name="actionSpeed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>速度</string>
   </property>
  </action>
  <action name="actionCourse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>航向</string>
   </property>
  </action>
  <action name="actionType">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>目标类型</string>
   </property>
  </action>
  <action name="actionAttr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>属性</string>
   </property>
  </action>
  <action name="actionHeight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>高度</string>
   </property>
  </action>
  <action name="actionLable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>绘制标牌</string>
   </property>
  </action>
  <action name="actionTrkType">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>航迹类型</string>
   </property>
  </action>
  <action name="actionChannel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>通道号</string>
   </property>
  </action>
  <action name="actionDrawTrackLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>连线</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
